README.md

```diff
@@ -37,7 +37,7 @@ Common commands in the CLI that you’ll be using:
 
 - nitric down : Undeploy a previously deployed stack, deleting resources
 - nitric run : Run your project locally for development and testing
-- nitric stack new : Create a new Nitric stack
+- nitric stack new [stackName] [cloudName] [region] : Create a new Nitric stack
```

Comment:
Not sure if we should include region at the top level as its varies too much by cloud.